docs/api.md

```diff
@@ -670,6 +670,25 @@ Adds a function which would be invoked in one of the following scenarios:
 
 The function is invoked after the document was created but before any of its scripts were run. This is useful to amend JavaScript environment, e.g. to seed `Math.random`.
 
+An example of using preloading some functions from an external file would be following,
```

Comment:
"An example of overriding the `navigator.languages` property before the page loads:"